src/utils/getPackageJson.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+import path = require('path');
+import { access } from 'fs';
+import { promisify } from 'util';
+
+const accessAsync = promisify(access);
+
+export async function getPackageJson(appDir: string): Promise<Object> {
```

Comment:
I added that interface based on the package.json schema, but it might be a bit of an overkill now. Let me know what you think

Reply:
Wooooaahhhh yeah I just meant a simple internal interface haha (`types/index.d.ts` is considered "external" because users can access those types. Anything _not_ exported in `types/index.d.ts` is "internal". We don't export `WorkerChannel` or any types related to that)

And for our purposes, we just need the `type` property, not everything else

Reply:
Ooops my bad haha, thanks! So is there a centralized place for the "internal" types/interfaces? Not sure where to export the PackageJson interface from since it's used in multiple files

Reply:
There is no centralized place - we have interfaces in many different files. The source of the object and the types for it should be in the same file. In this case that means put it in `WorkerChannel.ts` because that file parses and provides the object

Reply:
Makes sense, thanks :)